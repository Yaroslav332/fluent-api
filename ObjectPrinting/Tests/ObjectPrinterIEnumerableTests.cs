using System;
using ApprovalTests;
using ApprovalTests.Reporters;
using NUnit.Framework;

namespace ObjectPrinting.Tests
{
    [UseReporter(typeof(DiffReporter))]
    [TestFixture]
    public class ObjectPrinterIEnumerableTests
    {
        [Test]
        public void Print_Should_CorrectlyDisplayIEnumerables()
        {
            var e = new Enumerics();
            var printer = ObjectPrinter.For<Enumerics>();
            Approvals.Verify(printer.PrintToString(e));
        }
    }
}