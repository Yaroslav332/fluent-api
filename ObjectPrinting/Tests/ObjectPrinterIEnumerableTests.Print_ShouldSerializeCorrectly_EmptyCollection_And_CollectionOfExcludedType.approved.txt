LibraryRecords
	ActiveUsers = Empty_Collection
	ReadersVisits = [
		KeyValuePair`2
			Value = 01.01.0001 0:00:00,
		KeyValuePair`2
			Value = 03.03.0003 0:00:00
		]
	BooksCollection = Collection_Of_ExcludedType
